devs/docs/es-backports.rst

```diff
@@ -199,7 +199,7 @@ should be crossed out as well.
 - [s] 0286d0a769e Move distance_feature query building into MFT (#60614) (#60846)
 - [s] f44c28b5951 Deprecate and ignore join timeout (#60872)
 - [x] 1f49e0b9d01 Fix testRerouteOccursOnDiskPassingHighWatermark (#60869)
-- [ ] 2f76c48ea71 Propagate forceExecution when acquiring permit (#60634)
+- [x] 2f76c48ea71 Propagate forceExecution when acquiring permit (#60634)
```

Comment:
Removal of `forceExecution` flag in `TransportWriteAction` is not patched as we skipped memory tracking patch where it was introduced https://github.com/elastic/elasticsearch/commit/1ef2cd7f1ad90de9d0d2b8822dcda4749ecb56a5#diff-db0f68c49b24762cd94721808eb9c3c4c43720f0e31888434af5633154975e39R63 (renamed in https://github.com/elastic/elasticsearch/commit/68d56fa7dba83b8c40ddd0fb2604a581d8d8f955#diff-db0f68c49b24762cd94721808eb9c3c4c43720f0e31888434af5633154975e39)

Reply:
Do you know why we skipped the https://github.com/elastic/elasticsearch/commit/1ef2cd7f1ad90de9d0d2b8822dcda4749ecb56a5#diff-db0f68c49b24762cd94721808eb9c3c4c43720f0e31888434af5633154975e39R63 ? Looks like it's left out and not skipped.

Reply:
@mfussenegger ^^

Reply:
```
[ ] 1ef2cd7f1ad Add memory tracking to queued write operations (#58957)
```

Reply:
see https://github.com/crate/crate/pull/13334#discussion_r1039726448

Reply:
I skipped it in this PR, in https://github.com/crate/crate/pull/13344/files#diff-22c5fdb5356d19db5a53688ba0c1c7972a109d06357936bb06998da5d3ae341fR279

Reply:
Thanks got it.